sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.host.url=https://sonarcloud.io
+sonar.organization=sunbird-knowlg
+sonar.projectKey=sunbird-knowlg_knowlg-portal
+sonar.projectName=sunbird-knowledge
```

Comment:
Can't we use **sunbird-knowlg-portal**

Reply:
Yes @vinukumar-vs , We can use this name